doc/distributed-run.md

```diff
@@ -0,0 +1,52 @@
+## Running load test
+
+Starting a scenario on multiple nodes is slightly more difficult.
+You need a number of machines that will be actually running the test
+(*slaves*) and one controller machine (*master*, which might be one of the test nodes).
+Another aproach to do it is to use docker containers.
+
+Now instead of `amoc_local` use `amoc_dist` - this will tell amoc to distribute
+and start scenarios on all known nodes (except master).
+
+```erlang
+amoc_dist:do(my_scenario, 1, 100, Opts).
+```
+
+Start `my_scenario` spawning 100 amoc users with IDs from range (1,100) inclusive.
```

Comment:
> from the range (1,100)

Also, to be more mathematical, I'd say `[1,100]`. In maths, when describing an interval, parenthesis means that the end-points are not included, while square brackets means the endpoints are. But probably I'm the only idiot who cares about that 😂